requirements/docs.txt

```diff
@@ -1,6 +1,6 @@
-sphinx>=1.8,<=2.4.4
-sphinx-gallery>=0.7.0
-numpydoc>=1.0
+sphinx>=1.8,!=3.0.0,!=3.0.1,!=3.0.2,!=3.0.3,!=3.0.4
+numpydoc>=0.9
```

Comment:
Is there a reason for these changes here?

Reply:
numpydoc and sphinx-gallery that is

Reply:
No, I screwed up my rebase.

Reply:
yea.... that is always annoying.

Reply:
My goal over here is to build the doc with the last sphinx version, and if it appears to have broken links, to open a ticket upstream. I have no clue on this issue.